reconcile/aws_saml_idp/integration.py

```diff
@@ -127,6 +144,7 @@ def run(self, dry_run: bool, defer: Callable | None = None) -> None:
                 metadata=saml_idp_config.metadata,
             )
         working_dirs = ts.dump(print_to_file=self.params.print_to_file)
+        terraform_configurations = ts.terraform_configurations()
```

Comment:
this is only needed in `extended_early_exit_run`, can move there to avoid config dump when not needed

Reply:
Fixed in last commit